source/agora/utils/InetUtils.d

```diff
@@ -77,26 +90,7 @@ mixin template AddGetAllIPs()
 
         return ips;
     }
-}
-
-struct InetUtils
-{
-version (OSX)
-{
-    import core.sys.darwin.ifaddrs;
-    import core.sys.posix.netdb;
-    import core.sys.posix.netinet.in_;
-    import core.sys.posix.sys.socket;
 
-    mixin AddGetAllIPs;
-}
-version (linux)
-{
-    import core.sys.linux.ifaddrs;
-    import core.sys.posix.netdb;
-
-    mixin AddGetAllIPs;
-}
 version (Windows)
```

Comment:
Indentation is needed.

Reply:
That code wasn't changed though, but I'll add a commit.